sql-native/src/main/java/org/ballerinalang/sql/parameterprocessor/StatementParameterProcessor.java

```diff
@@ -0,0 +1,945 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.ballerinalang.sql.parameterprocessor;
+
+import io.ballerina.runtime.api.TypeTags;
+import io.ballerina.runtime.api.types.ArrayType;
+import io.ballerina.runtime.api.types.Field;
+import io.ballerina.runtime.api.types.StructureType;
+import io.ballerina.runtime.api.types.Type;
+import io.ballerina.runtime.api.utils.TypeUtils;
+import io.ballerina.runtime.api.values.BArray;
+import io.ballerina.runtime.api.values.BDecimal;
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BObject;
+import io.ballerina.runtime.api.values.BString;
+import io.ballerina.runtime.api.values.BXml;
+import org.ballerinalang.sql.Constants;
+import org.ballerinalang.sql.exception.ApplicationError;
+import org.ballerinalang.stdlib.io.channels.base.Channel;
+import org.ballerinalang.stdlib.io.channels.base.CharacterChannel;
+import org.ballerinalang.stdlib.io.readers.CharacterChannelReader;
+import org.ballerinalang.stdlib.io.utils.IOConstants;
+import org.ballerinalang.stdlib.time.util.TimeUtils;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.sql.Array;
+import java.sql.Clob;
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.RowId;
+import java.sql.SQLException;
+import java.sql.Struct;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.time.ZonedDateTime;
+import java.util.Iterator;
+import java.util.Locale;
+import java.util.Map;
+
+import static io.ballerina.runtime.api.utils.StringUtils.fromString;
+import static org.ballerinalang.sql.utils.Utils.throwInvalidParameterError;
+
+/**
+ * Reprsent the Process methods for statements.
```

Comment:
```suggestion
 * Represent the Process methods for statements.
 *
 * @since 0.5.6
 */
```

Reply:
I will change it

Reply:
I changed. commit id is e9dda1f10fc25a85b1a780c6ec2943325d527a5d

Reply:
I add the @since 0.5.6 line. commit id is 7882c5b2739775d889147d9e947381a42fa69a9f